docs/standard/serialization/how-to-chunk-serialized-data.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: chunk serialized data"
+description: You can chunk data to avoid problems with large data sets. Implement the IXmlSerializable interface to control serialization and deserialization.
```

Comment:
```suggestion
description: You can chunk data to avoid problems with large datasets. Implement the IXmlSerializable interface to control serialization and deserialization.
```